src/software/gui/visualizer/drawing/obstacle_artist.h

```diff
@@ -1,13 +1,13 @@
 #pragma once
 
-#include <QtWidgets/QGraphicsScene>
+#include "QtWidgets/QGraphicsScene"
```

Comment:
Curious about why we're making this change

Reply:
oops that is from a bazel error, fixed now